### PR TITLE
fix(userspace/libsinsp): reduce mem allocs for field extraction in chisels and formatting

### PR DESCRIPTION
**Pull request by jasondellaluce**
Signed-off-by: Jason Dellaluce <jasondellaluce@gmail.com>

**What type of PR is this?**

/kind bug

**Any specific area of the project related to this PR?**

/area libsinsp

**What this PR does / why we need it**:

This follows-up the changes of https://github.com/falcosecurity/libs/pull/368 and hunts down all the cases in which the unwanted allocation was present.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: userspace/chisel/chisel_api.cpp [hunk 1]
```diff
@@ -357,11 +357,11 @@ int lua_cbacks::field(lua_State *ls)
 		return 1;
 	}
 
-	vector<extract_value_t> rawvalues;
-	if(chk->extract(evt, rawvalues))
+	chk->m_extracted_values.clear();
+	if (chk->extract(evt, chk->m_extracted_values))
```
Comment (Andreagit97) [inline userspace/chisel/chisel_api.cpp:361]:
```suggestion
	if(chk->extract(evt, chk->m_extracted_values))
```

**Review by Andreagit97 — APPROVED** [review]
Thank you @jasondellaluce!
/approve

**Review by FedeDP — APPROVED** [review]
/approve

**Pull request merged**